### DISCUSSION TITLE
WEBSITE_RUN_FROM_PACKAGE + Functions App

### DISCUSSION THREAD

**Discussion by rajyraman** [Azure/azure-dev#2610] [Q&A]
Can azd deploy Functions that use WEBSITE_RUN_FROM_PACKAGE App Setting? This involves copying the Function code to storage account and setting the blob URL in WEBSITE_RUN_FROM_PACKAGE?

https://learn.microsoft.com/en-us/azure/azure-functions/run-functions-from-deployment-package#using-website_run_from_package--url

or, this is need hooks?

**Comment by rajeshkamal5050** [comment]
@wbreza added deploying form package feature [`azd deploy <service> --from-package <package_path>`](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/reference#options-11) as part of PR - https://github.com/Azure/azure-dev/pull/1833

Is this what you are looking for?

**Reply by rajyraman** [reply]
@rajeshkamal5050 - Doesn't seem to be it. I cannot find your Microsoft email. If you can share it I'll forward you the email I got from Matthew Henderson (MSFT) today regarding this topic of Function App deployment.
